src/pages/organism/navBar/molecules/cartBtn/atoms/CartText.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+
+class CartText extends Component {
+    render() {
+        const { cartDetails } = this.props;
+        if ( cartDetails.itemsCount === 0 ) {
+            return <div className='cartBtnText'>My Cart</div>
+        } else {
+            return  <div className='cartBtnText'>
+                <p>{cartDetails.itemsCount} items</p>
```

Comment:
We should extract itemsCount, itemsDiscountedPrice out of cartDetails before and passing it down, so that its redable.